From b87a077c2ef2eafc12f744e780cdb23a6924e3d2 Mon Sep 17 00:00:00 2001
From: David Sommerseth <davids@openvpn.net>
Date: Wed, 22 Jan 2025 23:44:14 +0100
Subject: [PATCH] tests: Only build journal-log-parse if systemd is present

This will fail to build on hosts without systemd development
headers and pkg-config support files installed.

Signed-off-by: David Sommerseth <davids@openvpn.net>
---
 src/tests/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/meson.build b/src/tests/meson.build
index 4d042e2..eb8cd59 100644
--- a/src/tests/meson.build
+++ b/src/tests/meson.build
@@ -273,7 +273,7 @@ executable('journal-log-parse',
     [
         'misc/journal-log-parse-test.cpp',
     ],
-    build_by_default: build_test_programs,
+    build_by_default: dep_systemd.found() and build_test_programs,
     link_with: [
         common_code,
     ],
